### PR TITLE
[Term Entry] Go Strings `ToLower()`

### PR DESCRIPTION
**Pull request by manviii27**
### Description

 Added term entry for Go String function `ToLower()` . File Location: `docs/content/go/concepts/string-functions/terms/tolower/tolower.md`

### Type of Change

- Adding a new entry

### Checklist

- [✓ ] All writings are my own.
- [✓ ] My entry follows the Codecademy Docs style guide.
- [ ✓] My changes generate no new warnings.
- [ ✓] I have performed a self-review of my own writing and code.
- [ ✓] I have checked my entry and corrected any misspellings.
- [ ✓] I have made corresponding changes to the documentation if needed.
- [ ✓] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ✓] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@manviii27 I made one small edit and this is good for a second review.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@manviii27 Just added one comment. Please update, and tag me when you've made the changes. Thanks

File: content/go/concepts/strings/terms/tolower/tolower.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+strings.ToLower(str)
+```
+
+Where `str` is the string to convert into lowercase.
+
+## Example
+
+The following example demonstrates the use of the `strings.ToLower()` function.
+
+```go
```
[the remark this answers is not in the document]
Comment (manviii27) [reply content/go/concepts/strings/terms/tolower/tolower.md:29]: @caupolicandiaz I've made the changes, please review. Thank you

**Review by caupolicandiaz — APPROVED** [review]
@manviii27 @SSwiniarski LGTM

**Pull request merged**